.github/scripts/build-mmtk.sh

```diff
@@ -3,5 +3,10 @@ set -xe
 . $(dirname "$0")/common.sh
 
 # simply build mmtk-v8 with nogc
-cd $THE_ROOT/mmtk
-rustup run $RUSTUP_TOOLCHAIN cargo build --features nogc
+rustup toolchain install $RUSTUP_TOOLCHAIN
```

Comment:
Perhaps change entire thing to 
```suggestion
rustup toolchain install $RUSTUP_TOOLCHAIN --target x86_64-unknown-linux-gnu  --component clippy rustfmt
```

Reply:
Done